docs/src/content/docs/guides/i18n.mdx

```diff
@@ -13,15 +13,20 @@ Starlight provides built-in support for multilingual sites, including routing, f
 
 1. Tell Starlight about the languages you support by passing [`locales`](/reference/configuration/#locales) and [`defaultLocale`](/reference/configuration/#defaultlocale) to the Starlight integration:
 
-   ```js {9-26}
+   ```js {8-26}
    // astro.config.mjs
    import { defineConfig } from 'astro/config';
    import starlight from '@astrojs/starlight';
 
    export default defineConfig({
    	integrations: [
    		starlight({
-   			title: 'My Docs',
+			// Set your siteTitle for multiple languages.
+   			title: {
+				en: 'My Docs',
+				'zh-cn': '我的文档',
```

Comment:
```suggestion
				'zh-CN': '我的文档',
```

The title object key must be BCP-47 tag e.g. "en", "ar", or "zh-CN" now. May be wo should normalize it for support it can be `zh-cn`、`pt-br`? @delucis